test/Stateless.Tests/AsyncActionsFixture.cs

```diff
@@ -559,15 +559,15 @@ public async Task FireAsyncTriggerWithParametersArray()
                     return Task.CompletedTask;
                 });
 
-            await sm.FireAsync(Trigger.X, 42, "Stateless", true, 420.69, Trigger.Y);
+            await sm.FireAsync(Trigger.X, 42, "Stateless", true, 123.45, Trigger.Y);
 
             Assert.Equal(expectedParam, actualParam);
         }
 
         [Fact]
         public async Task FireAsync_TriggerWithMoreThanThreeParameters()
         {
-            const string expectedParam = "42-Stateless-True-420.69-Y";
+            const string expectedParam = "42-Stateless-True-123.45-Y";
```

Comment:
The behavior of ToString is based on the current user's locale setting.
This test will fail on machines with a different locale.
One could either use ToString in the input as well or pin the locale.
Thread.CurrentThread.CurrentCulture = System.Globalization.CultureInfo.InvariantCulture;
string expectedParam = $"42-Stateless-True-{123.45}-Y";

Reply:
Good spot, @Fieslix, thank you. Should be resolved now.